content/posts/2025-02-26-whats_new_1_8.md

```diff
@@ -0,0 +1,295 @@
+
+The rest of the features are non-GA, meaning they should not be used in a production environment.  Network Events is a Technology Preview feature and the rest are Developer Preview features.
+
+1. Network Events
+2. eBPF Flow Filtering Enhancements
+3. UDN Observability
+4. ePBF Manager Support
+
+
+### Network Events
+
+With OVN-Kubernetes and Network Events, you can see what's happening with a packet, including why a packet was dropped or what network policy allowed or denied a packet from go through.  This gives you incredible insight into troubleshooting and is a long-awaited feature!
+
+On OVN-Kubernetes, this is disabled by default since this is a Technology Preview feature, so enable it by adding the feature gate named **OVNObservability**.  On the command line, enter `oc edit featuregate` and change the **spec** section to:
+
+```
+spec:
+  featureSet: CustomNoUpgrade
+  customNoUpgrade:
+    enabled:
+      - OVNObservability
+```
+
+<p style="text-align: center">Listing 3: Enable OVNObservability feature gate</p>
+
+This can take upwards of *10+ minutes* for this to take effect, so be patient.  Initially, it might seem like nothing has happened, but after about five minutes, you will lose connection to your cluster because OVN-Kubernetes has to restart.  Then after another five minutes, everything should be back online.
+
+To enable this feature in FlowCollector, enter `oc edit flowcollector` and configure the following in the **ebpf** section:
+
+```
+spec:
+  agent:
+    ebpf:
+      sampling: 1  # recommended so all flows are observed
+      privileged: true
+      features:
+      - NetworkEvents
+```
+
+<p style="text-align: center">Listing 4: Enable NetworkEvents feature in FlowCollector</p>
+
+A packet could be dropped for many reasons, but often times, it's due to a network policy.  In OCP, there are three different types of network policies.  Network Events supports all three.  Click the links to learn how to configure them.
+
+1. [Kubernetes Network Policy](https://docs.openshift.com/container-platform/4.18/networking/network_security/network_policy/about-network-policy.html) - similar to a firewall rule, typically based on an IP address, port, and protocol
+2. [Admin Network Policy](https://docs.openshift.com/container-platform/4.18/networking/network_security/AdminNetworkPolicy/ovn-k-anp.html) - allows cluster admin to create policies that are at the cluster level
+3. [Egress Firewall](https://docs.openshift.com/container-platform/4.18/networking/network_security/egress_firewall/viewing-egress-firewall-ovn.html) - controls what external hosts a pod can connect to
+
+In Figure 3, the Traffic flows table shows a column for Network Events.  The first row shows a drop from a network policy and the second row allowed the packet in an admin network policy.
+
+![Flows table with Network Events]({page.image('whats-new-1-8/flows-network_events.png')})
+Figure 3: Flows table with Network Events
```

Comment:
You could also mention / take a screenshot of the new network policies dashboards

Reply:
![Capture d’écran du 2025-02-07 10-45-22](https://github.com/user-attachments/assets/797ab2fc-e146-4087-88e9-e4cce8a3bbb3)